### PR TITLE
[pkg/spec] Update spec packaging for tflite layer

### PR DESCRIPTION
**Pull request by kparichay**
Update spec packaging for tflite layer with tflite dependency

**Self evaluation:**
1. Build test: [x]Passed [ ]Failed [ ]Skipped
2. Run test: [x]Passed [ ]Failed [ ]Skipped

Signed-off-by: Parichay Kapoor <pk.kapoor@samsung.com>

### CODE REVIEW

**Review by zhoonit — APPROVED** [review]
LGTM

File: packaging/nntrainer.spec [hunk 1]
```diff
@@ -103,6 +104,12 @@ Requires:	nnstreamer-tensorflow-lite
 Requires:	%{capi_machine_learning_inference}
 %endif # support_nnstreamer_backbone
 
+%if 0%{?support_tflite_backbone}
+BuildRequires: tensorflow-lite-devel
```
Comment (myungjoo) [inline packaging/nntrainer.spec:108]: In Tizen, I'd recommend tensorflow2-lite-devel

File: packaging/nntrainer.spec [hunk 2]
```diff
@@ -103,6 +104,12 @@ Requires:	nnstreamer-tensorflow-lite
...
+
+Requires:	tensorflow-lite
```
Comment (myungjoo) [inline packaging/nntrainer.spec:110]: There is no such packages in Tizen.

Comment (kparichay) [reply packaging/nntrainer.spec:110]: this is my bad. removed.

File: packaging/nntrainer.spec (hunk 1 above)
Comment (kparichay) [reply packaging/nntrainer.spec:108]: currently, nntrainer depends on tensorflow-lite and nnstreamer-tensorflow-lite, and tf-lite for android as well.
I will update both to tf2-lite for all together in a separate PR - raised #1244.

Comment (kparichay) [reply packaging/nntrainer.spec:108]: added #1245

**Review by lhs8928 — APPROVED** [review]
LGTM

**Pull request merged**